Tasks/CocoaPodsV0/cocoapods.ts

```diff
@@ -17,6 +17,21 @@ async function run() {
         tl.debug('Setting locale to UTF8 as required by CocoaPods');
         process.env['LC_ALL'] = 'en_US.UTF-8';
 
+        const proxyConfig: tl.ProxyConfiguration | null = tl.getHttpProxyConfiguration();
+        if (proxyConfig) {
+            const proxyUrl: URL = new URL(proxyConfig.proxyUrl);
+            let proxyAddress: string = `${proxyUrl.protocol}//${proxyUrl.host}`;
+            
+            if (proxyConfig.proxyUsername) {
+                proxyAddress = `${proxyUrl.protocol}//${proxyConfig.proxyUsername}:${proxyConfig.proxyPassword}@${proxyUrl.host}`;
```

Comment:
As an idea - could it make sense to add some method to the [task-lib](https://github.com/microsoft/azure-pipelines-task-lib) - to retrieve formatted proxy address like here?
Do we need to pass proxy bypass list also from proxyConfig - probably via 'no_proxy' variable?